deploy/cr.yaml

```diff
@@ -38,6 +38,7 @@ spec:
 #      runAsUser: 1001
 #      runAsGroup: 1001
 #      supplementalGroups: [1001]
+#    serviceAccountName: percona-xtradb-cluster-operator-workload
```

Comment:
Could we also add serviceAccountName into ProxySQL section?

Reply:
done